src/script/user/UserState.ts

```diff
@@ -39,10 +39,9 @@ export class UserState {
   public readonly isActivatedAccount: ko.PureComputed<boolean>;
   public readonly isTemporaryGuest: ko.PureComputed<boolean>;
   public readonly numberOfContacts: ko.PureComputed<number>;
-  public readonly self: ko.PureComputed<User>;
+  public readonly self = ko.observable<User | undefined>();
```

Comment:
another small improvements, is that `self` was re-computed everytime the `users` list would change. 
Which means that the entire app would re-render. 

Manually setting the `selfUser` at app start and then never update it again fixes this